sale_order_product_assortment/__manifest__.py

```diff
@@ -4,14 +4,18 @@
 {
     "name": "Sale Order Product Assortment",
     "summary": "Module that allows to use the assortments on sale orders",
-    "version": "15.0.2.0.0",
+    "version": "16.0.1.0.0",
     "development_status": "Beta",
     "category": "Sales",
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "maintainers": ["CarlosRoca13"],
     "license": "AGPL-3",
     "installable": True,
-    "depends": ["sale", "product_assortment", "base_view_inheritance_extension"],
+    "depends": [
+        "sale",
+        "product_assortment",
+        "web_domain_field",
```

Comment:
Don't use this one, but create a binary field and use it as the source for the domain.